source/confirm_conditions.md.erb

```diff
@@ -0,0 +1,35 @@
+---
+title: Confirm conditions
```

Comment:
This file needs to be renamed to `confirm_conditions.html.erb` (so with `html` file extension) in order for it to show up in the API docs.